irepela/homework_8/warrior.py

```diff
@@ -0,0 +1,39 @@
+# Details:  https://py.checkio.org/mission/the-warriors/share/e171c1a99e4e63b4551cfd820252edcd/
+
+
+class Warrior:
+
+    def __init__(self):
+        self.health = 50
+        self.attack = 5
```

Comment:
you could move health and attack to class level 
since they are immutable you do not need to worry when modify them on instance level